python/cudf/cudf/_lib/cpp/io/orc.pxd

```diff
@@ -31,8 +31,8 @@ cdef extern from "cudf/io/orc.hpp" \
 
         void set_columns(vector[string] col_names) except +
         void set_stripes(vector[vector[size_type]] strps) except +
-        void set_skip_rows(size_type rows) except +
-        void set_num_rows(size_type nrows) except +
+        void set_skip_rows(int64_t rows) except +
+        void set_num_rows(int64_t nrows) except +
```

Comment:
now that we allow higher values for nrows, do we check internally if we're reading more than size_type at a time?

Reply:
Yes we have that check in the reader. See https://github.com/rapidsai/cudf/pull/15094/files#diff-b6a7288b5d1b640bfd675a5fecd1bae49e4364b3d1024ad2bb6e3037f9beeb70R246-R250.